scipy/stats/_stats_py.py

```diff
@@ -889,6 +889,53 @@ def tsem(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
     return sd / np.sqrt(am.count())
 
 
+def rms(a, axis=0):
```

Comment:
Let's force this directly for the future 😀 
```suggestion
def rms(a, *, axis=0):
```